doc/langdef.md

```diff
@@ -617,8 +617,8 @@ evaluated:
     overload based on the types of the sub-expression values. See
     [Functions](#functions).
 
-Because CEL is free of side-effects, the order of evaluation among
-sub-expressions is not guaranteed. If multiple subexpressions would evaluate to
+Because CEL is free of side effects, the order of evaluation among
+sub-expressions is not guaranteed. If multiple sub-expressions would evaluate to
```

Comment:
I know it was inconsistent before your change, but I think unhyphenated "subexpression" is more commonly used. Would you mind updating to that?

Reply:
I don't have a strong opinion here, but currently `subexpression` is used in two places and `sub-expression` is used in twenty-one places within this document. Are you sure we should change the other twenty-one places instead?

Reply:
I'd prefer it, but don't want to hold up your PR over it.